packages/itwin/measure-tools/src/api/MeasurementViewTarget.ts

```diff
@@ -213,6 +232,11 @@ export class MeasurementViewTarget {
         return false;
     }
 
+    for (const viewId of this._viewIds) {
```

Comment:
Also need to check that they'Re the same size (see line 222)

Reply:
Opps! Nice catch